app/controllers/articles_controller.rb

```diff
@@ -1,4 +1,24 @@
 class ArticlesController < ApplicationController
+  # before_action :authenticate_user!, :except => [:show, :index]
```

Comment:
Please, don't forget before commit delete all comments. You can leave only comments that can help us understand the logic of your code

Reply:
Fixed